src/render.ts

```diff
@@ -133,6 +133,42 @@ export async function render(
       div.appendChild(storyResult);
       return true;
     }
+    case "Angular": {
+      const { props, component: StoryComponent, markup } = storyResult;
+      const { platformBrowserDynamic } = (await require("@angular/platform-browser-dynamic"));
+      const { Component, NgModule, destroyPlatform } = (await require("@angular/core"));
+      const { BrowserModule } = (await require("@angular/platform-browser"));
+      const {
+        imports = [],
+        declarations = [],
+        bootstrap = [],
+        providers = [],
+        schemas = [],
+      } = storyResult.moduleMetadata;
+
+      // Create a wrapper component to host the bindings
+      @Component({
+        selector: 'app-root',
+        template: markup,
+      })
+      class AppComponent {};
+      Object.keys(props).map((prop: string): string => AppComponent.prototype[prop] = props[prop]);
+
+      // Create the default module
+      @NgModule({
+        imports: [BrowserModule, ...imports],
+        declarations: [AppComponent, StoryComponent, ...declarations],
+        bootstrap: [AppComponent, ...bootstrap],
+        schemas,
+        providers,
+      })
+      class AppModule {};
+
+      // Init the app
+      div.insertAdjacentHTML('beforeend', '<app-root></app-root>');
```

Comment:
Sorry maybe I misunderstand this, but why adjacent ? Shouldn't we reuse the div already inplace ? the div where we want to render it ?

Reply:
Can't we have a selector for this particular div ?

Reply:
The `insertAdjacentHTML` method with the `beforeend` renders the string inside the div where it's invoked.

That div is the root, the one that holds the stories.

Reply:
Yes I know but I am wondering why we need another div behind the scene. I don't know how much is hard to have the correct selector for the App itself.